packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -164,12 +163,15 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
       // we rely on the volume in the cache for mapper. So we prefer actorUID if
       // it is defined, otherwise we use volumeId for the actor name.
       const uid = actorUID || volumeId;
-      volumeActors.push({ uid, volumeActor });
+      volumeActors.push({ uid, volumeActor, slabThickness });
     }
 
     this.addActors(volumeActors);
 
     if (immediate) {
+      // set the clipping planes
+      this.resetCamera(false, false);
```

Comment:
you are right, I moved the the reset of the camera into the addActors method. It makes more sense: i.e. everytime there is a new actor, it will add the clipping planes to the actors and it is independent from the request of rendering